<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnRead">
       <property name="text">
        <string>Read</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Append</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnWrite">
       <property name="text">
        <string>Write</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Filename</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LEFileName"/>
     </item>
     <item>
      <widget class="QComboBox" name="CMBfileFormat">
       <item>
        <property name="text">
         <string>.txt</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Output message</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="LEoutPut"/>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Editor</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="txtEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
